ui/packages/components/src/FunctionRunStatusIcon/RunStatusIcons.tsx

```diff
@@ -0,0 +1,23 @@
+import { type FunctionRunStatus } from '@inngest/components/types/functionRun';
```

Comment:
Nit: can you rename the file to `RunStatusIcon.tsx` to match the component name?

Reply:
I'll only do it when we delete the old one. This is currently in the same folder as the deprecated FunctionRunStatusIcon.tsx